src/components/research/graph-legend.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { css } from "@emotion/core";
+import { mq, BreakPoint } from "../../util/mq";
+
+type GraphLegendProps = {
+  labels: { color: string; label: string }[];
+  forceVertical: boolean;
+  variant: LegendVariant;
+};
+
+export enum LegendVariant {
+  PILL,
+  BRICK,
+}
+
+type GraphLegendMarkerProps = {
+  color: string;
+  variant: LegendVariant;
+};
+
+const variantStyles = {
+  [LegendVariant.PILL]: {
+    width: "1.5rem",
+    height: "0.375rem",
+    borderRadius: "3rem",
+    alignItems: "center",
+  },
+  [LegendVariant.BRICK]: {
+    width: "0.75rem",
+    height: "1.875rem",
+    borderRadius: "0",
+    alignItems: "inherit",
+  },
+};
+
+const GraphLegendMarker = ({ color, variant }: GraphLegendMarkerProps) => {
+  const legendMarkerStyles = css`
+    background-color: var(--${color});
+    min-width: ${variantStyles[variant].width};
+    height: ${variantStyles[variant].height};
+    border-radius: ${variantStyles[variant].borderRadius};
```

Comment:
Try declaring both variants as a CSS class on the `legendMarkerStyle`, and then you only need one `const legendMarkerStyles = css…` statement. I think the way emotion works, this means generating fewer blocks of CSS.